lib/elixir/lib/string.ex

```diff
@@ -874,7 +874,7 @@ defmodule String do
 
   `mode` may be `:default`, `:ascii` or `:greek`. The `:default` mode considers
   all non-conditional transformations outlined in the Unicode standard. `:ascii`
-  lowercases only the letters A to Z. `:greek` includes the context sensitive
+  uppercases only the letters A to Z. `:greek` includes the context sensitive
```

Comment:
```suggestion
  capitalizes only the letters A to Z. `:greek` includes the context sensitive
```

I understand if you don't want this. Leaving it here anyways!